test/test_utils.py

```diff
@@ -54,7 +54,7 @@ def evaluate_objective_on_list(x):
     # second call should be faster as result is retrieved from cache
     t0 = time.time()
     evaluate_objective_on_list(x)
-    assert (time.time() - t0) < (sleep_time / 5.0)
+    assert (time.time() - t0) < pytest.approx(sleep_time / 5.0, rel=0.1)
```

Comment:
hmm, travis doesn't seem to like this, and i'd also choose a different route: we should just increase the `sleep_time` a bit, maybe to 2s. this test fails as travis only gives us a single core and running multiple threads in it seems to slow everything down to the point where the effect of caching is not visible any more.

Reply:
Okay, I reverted the change. Miraculously, the CI seems to pass this test now. I created this ticket to refactor the test: #185